source/_components/namecheapdns.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+title: "NamecheapDNS"
+description: "Keep your namecheap dynamic DNS up to date"
+date: 2017-10-11 20:16
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: namecheap.png
+ha_category: Utility
+featured: false
+ha_release: 0.56
+---
+
+With the namecheapDNS component you can automatically update your dynamic DNS entry.
+
+<p class='note warning'>
+Namecheap only supports IPv4 adresses to update.
+</p>
+
+To use the component in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+namecheapdns:
```

Comment:
Rename this to `subdomain` please. This looks like a hack at the moment.

Reply:
The component is called namecheapdns?

Reply:
Is there something to change or is this comment resolved cause the PR is already merged!?

Reply:
Sorry, wrong line. i was referring to `host`.

Reply:
At the namecheap doc site they reference to the parameter as host so i thought if you use the component in addition to the documentation from namecheap you will know what you have to fill in there.

The only thing i turned around was the "access_token" they reference it as "dynamic DNS Password"